docs/labels.md

```diff
@@ -2,8 +2,8 @@
 
 There are two main types of labels that kops can create:
 
-* `CloudLabels` become tags in AWS on the instances
-* `NodeLabels` become labels on the k8s Node objects
+* `CloudLabels` become tags in AWS on the ASGs(autoscaling groups) and are propagates to their instances.
+* `NodeLabels` become labels on the k8s Node objects and tags on the ASGs(with the prefix: k8s.io/cluster-autoscaler/node-template/label/) and are propagated to their instances.
```

Comment:
Maybe say: "`NodeLabels` become labels on the kubernetes Node objects.  To support cluster autoscaler, these are also set as tags on the ASGs (which the prefix`k8s.io/cluster-autoscaler/node-template/label/`) and are then also propagated to their instances." ?  Because otherwise the distinction is very subtle - the intention is that NodeLabels becomes labels on the nodes, and today we set them on the ASG but that's only because of the limitations of cluster-autoscaler.